pkg/virt-launcher/notify-client/client.go

```diff
@@ -270,7 +270,7 @@ func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEve
 		if interfaceStatus != nil || osInfo != nil {
 			event := watch.Event{Type: watch.Modified, Object: domain}
 			client.SendDomainEvent(event)
-			events <- event
+			updateEvents(event, domain, events)
```

Comment:
possible that we just need to remove this, and order stuff a bit
unless the 3 other occurrences can happen more than once and then we need to guard them as well

thanks @enp0s3 for the discussion

Reply:
as what David suggested, i will return the previous commit additionally
https://github.com/kubevirt/kubevirt/pull/4253#pullrequestreview-495694281

Reply:
Done